schema-engine/connectors/mongodb-schema-connector/tests/introspection/index/mod.rs

```diff
@@ -1249,10 +1248,14 @@ fn unsupported_types_in_an_index() {
 
     expected.assert_eq(res.datamodel());
 
-    res.assert_warning_code(3);
-    res.assert_warning(
-        "These fields are not supported by the Prisma Client, because Prisma currently does not support their types.",
-    );
+    let expect = expect![[r#"
+        *** WARNING ***
+
+        These fields are not supported by the Prisma Client, because Prisma currently does not support their types:
+          - model: A, field: data, type: JavaScriptCode
```

Comment:
Not sure what is this "JavaScriptCode" 🤔

Reply:
Looks like it comes from Bson https://github.com/search?q=repo%3Aprisma%2Fprisma-engines%20JavaScriptCode&type=code